src/candidate/CandidateDetail.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, Text, View } from 'react-native';
+import { Avatar, Icon } from 'react-native-elements';
+
+const CandidateDetail = props => (
```

Comment:
I don't see the need for a wrapper component like this. As far as I can tell, its only function is to destructure the summary prop